COPYRIGHT.md

```diff
@@ -0,0 +1,44 @@
+### PistonDevelopers COPYRIGHT Notice
+
+By contributing to this project after
+
+    29-05-2016
+
+you lend your vote to the majority of
+
+    95%, rounded up to nearest whole person
+
+of the ACTIVE CONTRIBUTORS to apply a future license change.
+
+ACTIVE CONTRIBUTOR is contributor who responds to a request for license
+change within a time limit of 90 days (approximately 3 months).
+
+Default votes apply for contributions before this date.
+
+Here is a diagram explaining when copyright holder is for person or group:
+
+               _____________________________________________________________
+              | 100% agreement | > 95% agreement | > 3 months               |
+              |----------------|-----------------|--------------------------|
+default       | 1 per person   | 1 per person    | 1 per person             |
+              |----------------|-----------------|--------------------------|
+lend vote     | 1 per person   | 1 for group     | if 100% agreement:       |
```

Comment:
Nope, still borked. I think `default` and `lend vote` in front of the table mess it up.